services/net/content/ContentManager.cs

```diff
@@ -374,8 +375,18 @@ private async Task HandleMessageAsync(ConsumeResult<string, SourceContent> resul
             }
 
             if (updateSourceContent && (existingContentId != null)) {
-                // before updating, reinstate the Id value
-                content.Id = existingContentId.Value;
+                // before saving, reinstate some values from the original content object
+                if (originalContent != null) {
```

Comment:
Are these the only properties that need to be reset?

Reply:
all of the other properties are being newed up from the content that came in from Kafka